dependencies.list

```diff
@@ -1,2 +1,2 @@
 PACKAGE_NAME=realm-core
-VERSION=5.10.3
+VERSION=5.11.0
```

Comment:
Shouldn't this just be 10.4?

Reply:
I guess you can argue both ways. Let's just get it out now.

Reply:
It is a new feature.